client/tests/shared/utils.test.ts

```diff
@@ -0,0 +1,161 @@
+import {
+  applyUserAssumptionsOverDefaults,
+  applyUserCostInputsOverDefaults,
+  getRestorationYearlyBreakdown,
+  transformAssumptionsData,
+} from "@shared/lib/utils";
+
+describe("shared/lib/utils", () => {
+  describe("transformAssumptionsData", () => {
+    const mockAssumptions = {
+      valid: [
+        { name: "Baseline reassessment frequency", value: "5" },
+        { name: "Buffer", value: "0.2" },
+        { name: "Carbon price increase", value: "0.03" },
+      ],
+      invalid: [
+        { name: "Invalid name", value: "10" },
+        { name: "Buffer", value: undefined },
+      ],
+    } as const;
+
+    it("should return empty object for empty input", () => {
+      const result = transformAssumptionsData([]);
```

Comment:
on a nutshell, this is right. but I guess there is no possible scenario in the platform that this can/should happen right?